### ISSUE TITLE
[A11Y] Missing heading tag in the "Editorial" text

### ISSUE THREAD

**Issue by Devika008** [pkp/pkp-lib#9351]
## Issue Description
// What is the nature of the issue? How does it impact users?

n the "Begin Your Submission" introductory page while selecting the section of the journal in which the submission appears, the text shows an element in bold that should be a heading level 2 in the interface.

## How to reproduce / Current behavior
// Step-by-step instructions explaining how to reproduce the problem. The environment required, explicit identification of the component in question

1. Create a new submission
2. You will encounter the problem on the first page when you click the "New Submission" button while selecting the section in which needs to be published

## What application are you using?
 OJS, OMP or OPS version X.X.X
 
 OJS 3.4

## Stack used 

1. Device: Laptop
2. OS: MacOS (Monterey)
3. Browser: Chrome
4. Version: 117

## Testing method

- Manual (keyword)
- Screen Reader (VoiceOver)

## Action
// Describe action that can be taken to remediate the issue. 

Adjust the markup to show the first and bold styled element as a heading level 2 (H2). When we do this, we will meet the WCAG Criteria of [2.4.6 Headings and Labels - AA](https://www.w3.org/WAI/WCAG21/Understanding/headings-and-labels.html)

## Additional information
// Please add any screenshots, code snippets, or other information we can use to investigate this issue.

![image](https://github.com/pkp/pkp-lib/assets/112014397/1d20ad78-2ddc-4a88-b720-14edc38b17b7)
